_posts/2022/2022-01-31-infrastructure-software-we-use-to-run-our-saas.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "Software & Infrastructure we use to Run Our SaaS"
+description: "If you do a lot with a small team, you have to pay for softare"
+layout: article
+category: ruby
+image:
+  base: "2022/infrastructure-software-we-use-to-run-our-saas"
+  alt: "Frame of a building under construction"
+  credit: "Jacek Dylag"
+  source: "https://unsplash.com/photos/nhCPOp4A2Xo"
+
+---
+
+Our team at [CoverageBook](https://coveragebook.com/) (& [AnswerThePublic](https://answerthepublic.com/)) do _a lot_ with a _very_ small number of developers and designers. Over 4,000 paying customers & 600k monthly vistors (many using a free version of our product) all with a product team of five, including me.
+
+One of the key principles we use to keep our lives calm, and reduce the stress of supporting so much with so few folks, is to buy a lot of software to support our products and reduce the burden on us.
+
+This ends up with _lots_ of paid software to augment our team in critical areas.
+
+We resist building ourselves and pick the simplest, most pleasing-to-use software for the job. We even try, wherever possible not to configure away from the default settings!
+
+Unless I specifically mention it, you can take our usage of these products as an implicit recommendation. Not that there aren’t other alternatives, perhaps better for you, but this is what we’re mostly happy with.
+
+
+## Why?
+
+Even the most expensive infrastructure bills pale in comparison to the cost of hiring a human being to build and maintain that code.
+
+
+## Why not?
+
+I mean, if you’re Amazon or Google or Facebook or Apple or Twitter you might want to build to your unique scale and requirements. But then you’ll be hiring thousands of developers and you’re in a _very different_ world to me.
+
+This approach scales surprisingly well to teams an order of magnitude (or two) bigger than our team.
+
+You do often have to make choices that may not be to your ideal aesthetic of functional choice, but even in this “paid for” software world there are plenty of alternative approaches from which you can assess and pick your favourite.
```

Comment:
I feel like this paragraph could be written a lot more succinctly!

Reply:
```suggestion
When choosing software, rather than building it, you often have to make choices that may not be ideal functionally or aesthetically. There are generally multiple providers with alternative approaches that you can choose from.
```